File: 2017-10-DevconDavao/demo-2017-10-08-android/Demo/core/src/main/java/com/example/entities/Cat.java
package com.example.entities;

import java.util.UUID;

/**
 * Created by Dev on 09/10/2017.
 */

public class Cat {
    private UUID id;
    private String name;
    private String title;

    public Cat(UUID id, String name) {
        this.id = id;
        this.name = name;
    }

    public UUID getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String grandioseName() {
        if (title != null && !title.trim().equals(""))
            return name + " '" + title + "'";

        return name;
    }

    public boolean isImportant() {
        return grandioseName().length() < 10;
    }
}

File: 2018-01-PSITS-SultanKudarat/practice-2018-01-13/Layered/Cat.cs
using System;

namespace Layered
{
    public class Cat
    {
        public Cat(Guid id)
        {
            Id = id;
        }

        public Guid Id { get; private set; }
        public string Name { get; set; }
        public string Title { get; set; }

        public string GrandioseName
        {
            get
            {
                return string.Format("{0}, \"{1}\"", Name, Title);
            }
        }

        public bool IsImportant
        {
            get
            {
                return GrandioseName.Length <= 10;
            }
        }
    }
}